tests/cases/elements/toucharea.slint

```diff
@@ -21,6 +21,7 @@ TestCase := Rectangle {
             x: 3phx;
             width: 4phx;
             clicked => { touch3+=1; }
+            mouse-cursor: default;
```

Comment:
Yes it is.  This used not to work because it would generate `MouseCursor::default` in C++,  despite cbindgen was adding a _ to avoid the keyword collision.
Now i add the `_` both in cpp.rs and in the header file.

Reply:
Excellent, makes sense